content/docs/best-practices/best-practices.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Best practices to maximise efficiency
```

Comment:
`Best practices for managing large dataform projects`

Maybe? Maximizing efficiency doesn't mean much to me